_episodes_rmd/02-r-plotting.Rmd

```diff
@@ -676,17 +714,17 @@ This type of visualization makes it easy to compare the range and spread of valu
 ## Adding to plot objects
 _[Back to top](#contents)_
 
-In `ggplot2` you can store a plot in an object name and then add layers to the base plot. We're going to make a base plot with continent on the x axis and life expectancy on the y axis that we can add to, so we don't have to type out as much stuff:
+In `ggplot2` you can store a plot in an object name and then add layers to the base plot. We're going to make a base plot with continent on the x axis and lung cancer rates on the y axis that we can add to, so we don't have to type out as much stuff:
 
-```{r}
-base_plot <- ggplot(data = gapminder_1997) +
-  aes(x = continent, y = lifeExp)
+```{r base_plot}
+base_plot <- ggplot(data = smoking) +
+  aes(x = continent, y = lung_cancer_pct)
 base_plot
```

Comment:
I went back and forth about it. It does make things easier from a teaching perspective. Though I think it might be a good idea to write out on the board/easel what base_plot is so that learners can always remember what to expect.